docs/openapi/openapi.json

```diff
@@ -125,7 +125,18 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/Pet"
+                  "type": "object",
```

Comment:
consider making this a schema... like: https://github.com/transmute-industries/api.did.actor/blob/main/public/spec/schemas/DidResolutionResponse.yml

Reply:
hmm... also... we should not be editing JSON... we should be editing yaml and generating json.

Reply:
This is the output of `npm run preserve`, I did not hand-edit this.

Reply:
It looks like `swagger-cli bundle` optimizes by inlining refs that are only used once, which is the case for this DidResolutionResponse item.